### PR TITLE
MPDX-8056 - Tools - Initial Page

### PR DESCRIPTION
**Pull request by wjames111**
## Description

- Fix styles
- Fix links
- Add notifications
- [Jira ticket 8056](https://jira.cru.org/browse/MPDX-8056)

## Checklist:

- [x] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [x] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [x] I have requested a review from another person on the project

### CODE REVIEW

File: src/components/Tool/Home/ToolsListHome.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import {
```
Comment (canac) [inline src/components/Tool/Home/ToolsListHome.ts:1]: This looks pretty similar to what's in `ToolsListNav.ts`. Would it be possible to generate this file from `ToolsListNav.ts`? If that's not feasible, that's OK.

**Review by canac — APPROVED** [review]
Looks good! My only concern is some duplication in the tools list. It might be unavoidable; I wasn't sure.

File: src/components/Tool/Home/ToolsListHome.ts (hunk 1 above)
Comment (canac) [reply src/components/Tool/Home/ToolsListHome.ts:1]: Oh did you duplicate it to get a specific order for the tools? It might be OK then.

**Review by caleballdrin — APPROVED** [review]
Looks good to me!

File: src/components/Tool/Home/ToolsListHome.ts (hunk 1 above)
Comment (canac) [reply src/components/Tool/Home/ToolsListHome.ts:1]: I think you can keep it.

**Pull request merged**